hs_core/models.py

```diff
@@ -1358,6 +1358,26 @@ def remove(cls, element_id):
 def short_id():
     return uuid4().hex
 
+
+from mezzanine.pages.managers import PageManager
```

Comment:
Should this PageManager import be put on the top of the file together with other mezzanine imports?